.github/workflows/python-app.yml

```diff
@@ -69,4 +69,4 @@ jobs:
         POSTGRES_PASSWORD: postgres
         DATABASE_URL: postgresql://postgres:postgres@postgres:5432/postgres
       run: |
-        pytest
+        pytest --capture=tee-sys --cov=. 
```

Comment:
Can you help me understand why these parameters exist?

Reply:
* **`--capture=tee-sys`**:  This instructs pytest to capture standard output and error streams (stdout and stderr) while simultaneously "teeing" them to the console. This ensures you can see the output in real-time while enabling pytest to capture the logs for later analysis. By default, if this parameter is absent, pytest might only capture the output and not print it to the console, making it harder to follow along during test execution. 

* **`--cov=.`**: This enables code coverage reporting with the `pytest-cov` plugin. The `.` specifies that coverage should be calculated for the current directory. This helps you determine what percentage of the code is being exercised by the tests, allowing to identify areas where we might need to write more tests. Without this parameter, pytest wouldn't generate a code coverage report, making it difficult to gauge how well the tests cover the codebase. Please see [this issue](https://github.com/waleedkadous/ansari-backend/issues/23).